Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -293,7 +281,7 @@ namespace xr
 
             // Create the ARCore ArSession
             {
-                ArStatus status = ArSession_create(hmdImpl.Env(), hmdImpl.AppContext(), &session);
+                ArStatus status = ArSession_create(GetEnvForCurrentThread(), GetAppContext(), &session);
```

Comment:
Did you test this? Seems right to me, but I want to make sure this doesn't break things for @Alex-MSFT.

Reply:
Yes, though you just reminded me to make sure to check that the thread-local detach is actually getting called.

Reply:
Confirmed that detach is getting called.